pyside2kit/ps2kit.py

```diff
@@ -290,9 +291,17 @@ def __init__(self, button_label="Browse", title="Select", root_folder=os.getcwd(
             self.root_folder = root_folder
 
         self._browse_layout = QtWidgets.QHBoxLayout()
-        self._path_line_edit = QtWidgets.QLineEdit(parent=self)
+
+        object_name_path = ''
+        object_name_button = ''
+        if objectName:
+            object_name_path    = objectName + '_QLineEdit'
+            object_name_button  = objectName + '_QPushButton'
```

Comment:
since we cant assign duplicate objectnames, and this widget contains 2 qwidgets, we gave them the same name and add a suffix to the end